packages/legend-studio/src/components/editor/edit-panel/ModelLoader.tsx

```diff
@@ -187,25 +214,35 @@ export const ModelLoader = observer(() => {
             </button>
           )}
         </div>
-        <div className="model-loader__header__action">
-          <button
-            className="btn--dark model-loader__header__load-btn"
-            onClick={loadModel}
-            disabled={modelLoaderState.isLoadingModel}
-            tabIndex={-1}
-            title="Load model"
-          >
-            Load
-          </button>
-        </div>
+        {!modelLoaderState.currentExtraNativeInputType && (
+          <div className="model-loader__header__action">
+            <button
+              className="btn--dark model-loader__header__load-btn"
+              onClick={loadModel}
```

Comment:
I think we also need an extension when user clicks `load`. Your extension will handle that click as load/import your model to studio.